tests/e2e/support/commands.js

```diff
@@ -91,6 +99,11 @@ Cypress.Commands.add('login', (options = {}, route, isMockingExternalRequests =
   cy.openPopup(async (contentWindow) => {
     /* eslint-disable-next-line no-param-reassign */
     contentWindow.localStorage.state = JSON.stringify(await getLoginState(options));
+
+    if (options.pendingTransaction) {
+      // eslint-disable-next-line no-param-reassign
+      contentWindow.localStorage['sh-wallet_transactions_state_ae_uat'] = JSON.stringify(preparePendingTransactionToLocalStorage(options.pendingTransaction));
+    }
```

Comment:
In my opinion this is rather bad that the test case is creating its own local storage state. I think our logic should handle the lack of pending transactions in the local storage.

Maybe you could use `addCustomPendingTransactionForAccount` method instead?

Reply:
Original test worked on localStorage. What do you think @CedrikNikita @subhod-i ?